changelog.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Modelization feature:
   * Allow to use the mouse coordinates when placing new component on drag and drop feature.
 * Add new environment variable `KEEP_CYPRESS_ATTRIBUTE` to keep all cypress attribute in html.
+* Expand model folders and open parsable files.
```

Comment:
- Move this in `Text editor feature`
- change it by something like this:
`On open text editor open all related files of the current model`
- Add in `### Fixed` section the bug fix you do

Reply:
done